pkg/dto/aerospike_cluster.go

```diff
@@ -59,6 +64,26 @@ func (a *AerospikeCluster) Validate() error {
 	return nil
 }
 
+// validateSeedNodesTLSConsistency ensures that if any seed node has TLS configuration,
+// then all seed nodes must have TLS configuration.
+func (a *AerospikeCluster) validateSeedNodesTLSConsistency() error {
+	var hasTLSNodes, hasNonTLSNodes bool
```

Comment:
```suggestion
	if len(a.SeedNodes) <= 1 {
		return nil
	}
	
	var hasTLSNodes, hasNonTLSNodes bool
```
To optimize for a single seed node configuration.